### PR TITLE
Reorder and make the sidebar labels prettier

### PR DESCRIPTION
**Pull request by Gnuxie**
image

This will be followed up by moving the moderator's guide out of the bot section entirely, and breaking it up into something more palatable.

### CODE REVIEW

File: docs/contributing/CONTRIBUTING.md [hunk 1]
```diff
@@ -83,8 +87,7 @@ Go ahead and edit anything.
 ## Making Pull Requests
 
 The preferred and easiest way to contribute changes to Matrix is to fork the
```
Comment (FSG-Cat) [inline docs/contributing/CONTRIBUTING.md:89]:
```suggestion
The preferred and easiest way to contribute changes to Matrix is to fork the
```

To matrix????

File: docs/appservice/_category_.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "label": "Appservice",
-  "position": 1,
```
Comment (FSG-Cat) [inline docs/appservice/_category_.json:3]: This was Tbh because Alphabetical order looks nice. But it makes sense to change it.

**Review by FSG-Cat — COMMENTED** [review]
This PR looks fine after the Question is answered and also i have feedback about the next PR that is planed based on this due to that i personally think the way you describe the split of the mod guide is too extreme.

File: docs/contributing/CONTRIBUTING.md (hunk 1 above)
Comment (Gnuxie) [reply docs/contributing/CONTRIBUTING.md:89]: Open an issue, this is text from Mjolnir.

File: docs/appservice/_category_.json (hunk 2 above)
Comment (Gnuxie) [reply docs/appservice/_category_.json:3]: Yeah i don't want newbies reading the appservice first

**Review by FSG-Cat — APPROVED** [review]
Concerns Resolved or moved to separate future issue and PR.

**Pull request merged**